package.json

```diff
@@ -47,7 +49,7 @@
     "regenerator-runtime": "^0.13.7",
     "text-encoding": "^0.7.0",
     "typescript": "^4.3.5",
-    "vite": "^2.6.14",
+    "vite": "^2.9.1",
```

Comment:
Now with the recent theming updates, it added a custom PostCSS plugin which doesn't play well with `vite@2.9.0`+, see https://github.com/vector-im/hydrogen-web/issues/722

We will probably need to resolve that situation before merging this PR.

Reply:
This was fixed in Vite via https://github.com/vitejs/vite/issues/7822 ->https://github.com/vitejs/vite/pull/7827 and is part of `vite@2.9.8` 🎉

Reply:
Updated in PR